src/js/helpers/redirectHelper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * redirectHelper.js
+ * Created by Lizzie Salita 2/23/18
+ **/
+
+import storeSingleton from 'redux/storeSingleton';
+import * as redirectModalActions from 'redux/actions/redirectModal/redirectModalActions';
+
+
+export const showRedirectModal = (url) => {
+    storeSingleton.store.dispatch(redirectModalActions.showModal(url));
+};
+
+export const hideRedirectModal = () => {
```

Comment:
Do we actually need this? Isn't the only way to close the modal from inside the modal (which has access to `hideModal` via `react-redux`)?

Reply:
Removed `hidRedirectModal`